docs/README.md

```diff
@@ -1,4 +1,4 @@
-# Kubernetes Documentation: HEAD
+# Kubernetes Documentation: releases.k8s.io/HEAD
```

Comment:
Did you mean this to be linky? 'Cuz it's raw text right now, which looks a little odd.